src/include/daos/dtx.h

```diff
@@ -39,10 +39,14 @@ struct daos_tx_id {
 };
 
 static inline void
-daos_generate_dti(struct daos_tx_id *dti)
+daos_generate_dti(struct daos_tx_id *dti, bool zero)
```

Comment:
style: to be consistent with code convention, it should be called daos_dti_generate, I actually suggest to rename it to daos_dti_init, because it generates nothing if server side transaction is disabled right?

Also, do we actually want to pass in "zero" at here? or we could move it back to .c, and read the tunable directly.

Reply:
The client still will generate the DTI even if server disable DTX, such DTX will not be ignored directly if server disabled the DTX. The "zero" is used for the case that client disabled the IO dispatch.